src/mmw/js/src/draw/tests.js

```diff
@@ -78,7 +79,7 @@ describe('Draw', function() {
 
             var $li = $($el.find('#select-area-region li a').get(0));
             $li.trigger('click');
-
+            App._mapView._leafletMap.fireEvent('click', {'latlng': [29.979, 31.134]});
```

Comment:
Consider replacing with `App.getLeafletMap`. By convention, properties with underscore prefix aren't meant to be accessed outside of their modules.